packages/wallets/src/features/cashier/modules/WithdrawalCrypto/utils/withdrawalCryptoValidators.ts

```diff
@@ -0,0 +1,82 @@
+import { TWithdrawalCryptoContext } from '../provider';
```

Comment:
that file probably should have tests, and then tests validating that those validators are actually being used by the form, 

or just one file testing all the cases within the form test itself ;)